app/Http/Controllers/MismatchController.php

```diff
@@ -20,8 +19,22 @@ class MismatchController extends Controller
      */
     public function index(MismatchesRequest $request)
     {
-        $mismatches = Mismatch::whereIn('item_id', $request->ids)->get();
+        $query = Mismatch::whereIn('item_id', $request->ids);
 
-        return MismatchResource::collection($mismatches);
+        // limit to 'pending',
+        // unless include_reviewed parameter is provided
+        if (!$request->include_reviewed) {
+            $query->where('status', 'pending');
+        }
+
+        // limit to non-expired,
+        // unless include_expired parameter is provided
+        if (!$request->include_expired) {
+            $query->whereHas('importMeta', function ($import) {
+                $import->where('expires', '>=', Carbon::now());
```

Comment:
```suggestion
                $import->where('expires', '>=', now());
```

I don't think we actually need to use Carbon directly here, as carbon is not a direct dependency of our app and  Laravel provides a [global `now()` helper](https://laravel.com/docs/8.x/helpers#method-now) for this use case. This will also help with ["stopping time" in tests](https://laravel.com/docs/8.x/mocking#interacting-with-time) (See #15 for a usage example in our repo).